.gitignore

```
@@ -29,9 +29,10 @@ yarn-error.log
 /public/env-config.js
 env-config.js
 .idea/
-/test-results/
-/playwright-report/
-/playwright/.cache/
+# TODO: delete unnecessary stuff
+test-results/
+playwright-report/
+playwright/.cache/
 playwright/.auth/
 .npmrc
 
```

- Also add your script

```suggestion
packages/devtron-fe-lib
```

---

fe-lib PR

---

Added the script to devtron-fe-lib. But the script should be run from the root of monorepo